src/modules/guideline-service/GuidelineServiceController.ts

```diff
@@ -1,12 +1,95 @@
-import { Router } from "express";
+import { Router, Request } from "express";
+import proxy = require("express-http-proxy");
 import { Controller } from "../../interfaces/Controller";
 
+const OUTCOME_API = process.env.OUTCOME_API || 'localhost:3000';
```

Comment:
I think outcome suggestion service runs on 9000.

Reply:
My .env says 7000, 9000 is utility service.  So I'm using 7000 instead.

Reply:
Okay cool. I wasn't sure about the port number but 3000 sounded off.